### DISCUSSION TITLE
"Microservices" becomes "Workload" in the Merge Modal

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#70] [Q&A] [answered]
Why the "Microservices" section is renamed to "Workload" in the Merge Branch modal?
It confused me.

**Comment by davidebianchi** [comment, marked as answer]
The name in the merge modal group all the menù item under the subgroup Workload of the sidebar. So, in the Workload section you will find also CronJobs and replicas, and not only Microservices
